www/locales/en.json

```diff
@@ -76,9 +76,9 @@
       "ARJUN_DEV_JI": "Guru Arjun Dev Ji",
       "GOBIND_SINGH_SAHIB_JI": "Guru Gobind Singh Ji",
       "GRANTH_SAHIB_JI": "Guru Granth Sahib Ji",
-      "HAR_GOBIND_SAHIB_JI": "Guru Har Gobind Ji",
-      "HAR_KRISHAN_SAHIB_JI": "Guru Har Krishan Ji",
-      "HAR_RAI_SAHIB_JI": "Guru Har Rai Ji",
+      "HARGOBIND_SAHIB_JI": "Guru Hargobind Ji",
```

Comment:
nitpick: the key change was unnecessary.

Reply:
wanted to keep consistency